packages/client/src/components/App.tsx

```diff
@@ -1,7 +1,73 @@
 import '../styles/reset.css';
+import '../styles/tictactoe.css'
+import React from 'react';
+import { Difficulty, GameStatus, Marker, Player } from '@mapistry/take-home-challenge-shared';
 
-export const App = () => (
-  <main>
-    <div>FIXME: add your solution here!</div>
-  </main>
-);
+import { begin, move } from '../api'
+import { Square } from './Square'
+
+export class App extends React.Component<unknown, GameStatus> {
+  constructor(props: unknown) {
+    super(props)
+    this.state = {
```

Comment:
I'm curious if there is an idiomatic way to use the type/interface definition here to initialize state.